src/pages/ConnectWallet/ConnectWallet.tsx

```diff
@@ -62,6 +66,10 @@ const langSelectorMarginTop = {
 const metamaskIcon = <MetaMaskIcon />
 
 export const ConnectWallet = () => {
+  const { data: walletClient } = useWalletClient()
+
+  console.log({ walletClient })
```

Comment:
Confirming that we can get a viem `WalletClient`, which we can - from there on, just drill me down in hdwallet and leverage wagmi sane API 
<img width="374" alt="image" src="https://github.com/user-attachments/assets/bdc4f0b0-ec72-4719-8a4b-a916714e48a9">